From b7c0169a62d4e6ad1a0a9de4ef8ae8089ee98872 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sat, 11 Jun 2022 19:12:19 +0200
Subject: [PATCH] output: include stdarg.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes a build error with alsa-utils when build with a uClibc toolchain:

alsa-utils/host/x86_64-buildroot-linux-uclibc/sysroot/usr/include/alsa/output.h:75:66:
 error: unknown type name ‘va_list’
   75 | int snd_output_vprintf(snd_output_t *output, const char *format, va_list args);
      |                                                                  ^~~~~~~
alsa-utils/host/x86_64-buildroot-linux-uclibc/sysroot/usr/include/alsa/output.h:1:1:
 note: ‘va_list’ is defined in header ‘<stdarg.h>’; did you forget to ‘#include <stdarg.h>’?

Patch sent upstream: https://github.com/alsa-project/alsa-lib/pull/237

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 include/output.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/output.h b/include/output.h
index 4a970dc4..5e16420b 100644
--- a/include/output.h
+++ b/include/output.h
@@ -28,6 +28,8 @@
 #ifndef __ALSA_OUTPUT_H
 #define __ALSA_OUTPUT_H
 
+#include <stdarg.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
-- 
2.30.2

